app/jinja_filters.py

```diff
@@ -251,7 +251,7 @@ def should_wrap_with_fieldset_processor() -> dict[str, Callable]:
 
 def get_min_max_value_width(
     min_max: Literal["minimum", "maximum"], answer: AnswerType, default_value: int
-) -> int:
+) -> dict[str, dict[str, int]]:
```

Comment:
this was correct before

Reply:
Yeah but because I removed the cast to int I had to change this because the return from this is calling that function

Reply:
Ah! The error is caused by something else
when you do 
`schema = g.get("schema")` mypy doesn't know the type of schema, so schema then fails to pick up the type of `min_and_max_map` which is why the mypy error is now 
`Returning Any from a function declared to return int`

The return type was correct as `int`
but to fix the error you just need to do
`schema: QuestionnaireSchema = g.get("schema")`

Reply:
Got it cheers